pvlib/pvsystem.py

```diff
@@ -142,19 +142,18 @@ class PVSystem:
 
     module : string, optional
         The model name of the modules.
-        May be used to look up the module_parameters dictionary
-        via some other method.
 
-    module_type : string, default 'glass_polymer'
-         Describes the module's construction. Valid strings are 'glass_polymer'
-         and 'glass_glass'. Used for cell and module temperature calculations.
+    module_type : string, default `glass_polymer`
```

Comment:
I am not sure this formatting change is desirable for parameter default values.  The quotes indicate a string literal while backticks could suggest a variable name.

Reply:
I thought the single ` indicated a parameter value while double `` indicated a parameter. They render differently in the docs. Is this incorrect? I don't have any personal preference, I just thought this was the way to do it. Will revert to ' ' if the ` is misleading

Reply:
If it were just me, I don't think I'd ever use single backticks in sphinx/RST markup.  It makes sense to me to format parameter names, values, literals, etc as code (i.e. with double backticks).  I wildly speculate that we have single backticks in pvlib's codebase at least in part because of muscle memory from markdown (where single backticks render as code).

Of course, it is possible I forget or don't realize some good use of single backticks!

For the specific case of parameter default values, I think we should stick with no backticks at all, at least for now.

Reply:
Yes, I'd say those should use double backticks. But beware of the scope creep that @cwhanse mentioned :)